src/hooks/__mocks__/useAppConstants.ts

```diff
@@ -0,0 +1,81 @@
+import { AppConstants } from 'src/constants';
+
+export default function useAppConstants(type: string) {
+  // moved data declaration to the bottom for readability
+  // eslint-disable-next-line @typescript-eslint/no-use-before-define
+  return data[type];
+}
+
+const emptyRecord = {
+  id: '',
+  changed_at: '',
+  changed_by: '',
+  created_at: '',
+};
+
+export const data = {
```

Comment:
I added app constants in this file: https://github.com/digitalaidseattle/seattle-humane-app/blob/sprint-3/supabase/seed.sql.

Should we delete that?

Reply:
No, what you have in the seed is great. This file is just mock data for unit tests. Our tests should/will run without a live DB connection. This data is used by the mock version of the supabase client during test runs. (note the folder location is `__mocks__`. see (Jest manual mocks)[https://jestjs.io/docs/manual-mocks])